### PR TITLE
added summary of output

### PR DESCRIPTION
**Pull request by am-kaiser**
branch includes scripts to create a short and detailed summary of the checker output

### CODE REVIEW

**Review by atmodatcode — APPROVED** [review]

File: atmodat_data_checker/result_output/output_directory.py [hunk 1]
```diff
@@ -0,0 +1,40 @@
...
+
+import os
+from datetime import datetime
+from pathlib import Path
+
+global opath, opath_base
+
+# Get current date and time
+now = datetime.now()
+now_formatted = now.strftime("%Y%m%d_%H%M") + '/'
```
Comment (jkretz) [inline atmodat_data_checker/result_output/output_directory.py:11]: I don't like the output of the checkers to be placed in a directory labeled with the date. It might be a little hard to follow what results are in which directory if you do checks on multiple files and directories. So why not label it after the name of the file or the directory the files to be checked are in?

File: atmodat_data_checker/result_output/output_directory.py [hunk 2]
```diff
@@ -0,0 +1,40 @@
...
+
+    return file_names
+
+
+def create_directories():
+    # Types of checks to be performed
+    check_types = ["mandatory", "recommended", "optional", "CF"]
+    # Create directory to store output from checker if it does not exist already
+    if not os.path.isdir(opath):
+        os.system('mkdir -p ' + opath)
```
Comment (jkretz) [inline atmodat_data_checker/result_output/output_directory.py:38]: This won't work on Windows machines. To be platform-independent use `os.mkdir(opath)`

File: atmodat_data_checker/result_output/output_directory.py [hunk 3]
```diff
@@ -0,0 +1,40 @@
...
+        for check in check_types:
+            os.system('mkdir -p ' + opath + '/' + check)
```
Comment (jkretz) [inline atmodat_data_checker/result_output/output_directory.py:40]: Same here

File: atmodat_data_checker/result_output/summary_creation.py [hunk 4]
```diff
@@ -0,0 +1,130 @@
+"""module to create summary of results from output of atmodat data checker"""
+
+import json
+import os
+
+import pandas as pd
```
Comment (jkretz) [inline atmodat_data_checker/result_output/summary_creation.py:6]: Pandas is not part of the modules that are installed in the processes of setting up the Atmodat checkers. This should therefore mentioned in the requirements

**Review by jkretz — CHANGES_REQUESTED** [review]
Everything looks okay to me in principle. I attached some parts that should be revised.
Furthermore, I have a hard time to interpret the output of the `detailed_summary.csv` as there is quite a lot information, especially in the `msgs` column, which is not that nicely formatted (at least when openiing it with Libre Office).
Before merging, I would recommend to squash all commits into one.

File: atmodat_data_checker/result_output/output_directory.py (hunk 1 above)
Comment (atmodatcode) [reply atmodat_data_checker/result_output/output_directory.py:11]: I will think about this and revise it, but for now, I will keep it for testing the new summary module.
